docs/tutorials/trainers.ipynb

```diff
@@ -161,7 +161,10 @@
    "source": [
     "task = RegressionTask(\n",
     "    model=\"resnet18\",\n",
+    "    weights=None,\n",
     "    pretrained=True,\n",
```

Comment:
Should we also drop pretrained=True?

Reply:
I think so yes. There was a discussion somewhere of weights being one of `"imagenet", None or WeightEnum`? So is that what you had in mind?

Reply:
See https://torchgeo.readthedocs.io/en/latest/api/trainers.html#torchgeo.trainers.RegressionTask:

> **weights** – Either a weight enum, the string representation of a weight enum, True for ImageNet weights, False or None for random weights, or the path to a saved model state dict.